static/js/components/RunSummary.jsx

```diff
@@ -0,0 +1,423 @@
+import IconButton from '@material-ui/core/IconButton';
+import Grid from '@material-ui/core/Grid';
+import BuildIcon from '@material-ui/icons/Build';
+
+import Link from '@material-ui/core/Link';
+import PictureAsPdfIcon from '@material-ui/icons/PictureAsPdf';
+
+import Menu from '@material-ui/core/Menu';
+import MenuItem from '@material-ui/core/MenuItem';
+
+import MUIDataTable from "mui-datatables";
+import ThumbnailList from "./ThumbnailList";
+import styles from './RunSummary.css';
+import { observingRunTitle } from './AssignmentForm';
+import { ObservingRunStarList } from './StarList';
+import * as SourceAction from '../ducks/source';
+import * as Action from '../ducks/observingRun';
+import { ra_to_hours, dec_to_hours } from "../units";
+
+const VegaPlot = React.lazy(() => import('./VegaPlot'));
+const AirmassPlot = React.lazy(() => import('./AirmassPlot'));
+
+
+const SimpleMenu = ({ assignment }) => {
+  const [anchorEl, setAnchorEl] = React.useState(null);
+  const dispatch = useDispatch();
+
+  const handleClick = (event) => {
+    setAnchorEl(event.currentTarget);
+  };
+
+  const handleClose = () => {
+    setAnchorEl(null);
+  };
+
+  const ispending = assignment.status === "pending";
+  const isdone = assignment.status === "complete";
+  const isnotdone = assignment.status === "not observed";
+
+  const completeAction = () => {
+    handleClose();
+    return dispatch(
+      SourceAction.editAssignment({ status: "complete" }, assignment.id)
+    );
+  };
+
+  const pendingAction = () => {
+    handleClose();
+    return dispatch(
+      SourceAction.editAssignment({ status: "pending" }, assignment.id)
+    );
+  };
+
+  const notObservedAction = () => {
+    handleClose();
+    return dispatch(
+      SourceAction.editAssignment({ status: "not observed" }, assignment.id)
+    );
+  };
+
+  return (
+    <div>
+      <IconButton
+        aria-controls="simple-menu"
+        aria-haspopup="true"
+        onClick={handleClick}
+        variant="contained"
+      >
+        <BuildIcon />
+      </IconButton>
+      <Menu
+        id="simple-menu"
+        anchorEl={anchorEl}
+        keepMounted
+        open={Boolean(anchorEl)}
+        onClose={handleClose}
+      >
+        {
+          (ispending || isnotdone) && (
+          <MenuItem onClick={completeAction} variant="contained" key={`${assignment.id}_done`}>
+            Mark Observed
+          </MenuItem>
+          )
+}
+        {
+          (ispending || isdone) && (
+          <MenuItem onClick={notObservedAction} variant="contained" key={`${assignment.id}_notdone`}>
+            Mark Not Observed
+          </MenuItem>
+          )
+}
+        {
+          (isdone || isnotdone) && (
+          <MenuItem onClick={pendingAction} variant="contained" key={`${assignment.id}_pending`}>
+            Mark Pending
+          </MenuItem>
+          )
+}
+        {isdone && (
+        <MenuItem key={`${assignment.id}_upload_spec`} onClick={handleClose}>
```

Comment:
Can you help me understand what's supposed to happen here? The menu item says "Upload Spectrum" but its onClick is just tied to handleClose. Perhaps there's supposed to be a `Link` here as below with Upload Phot

Reply:
This is just a placeholder until the upload spectrum page is done.

Reply:
I can take this out if you think it shouldn't be integrated yet. Or I can comment it out.

Reply:
How about just change the display text to include "Coming soon" or something, so it's clear it's not supposed to work yet?

Reply:
Done